src/core/public/chrome/chrome_service.tsx

```diff
@@ -116,13 +118,24 @@ export class ChromeService {
     const helpSupportUrl$ = new BehaviorSubject<string>(KIBANA_ASK_ELASTIC_LINK);
     const isNavDrawerLocked$ = new BehaviorSubject(localStorage.getItem(IS_LOCKED_KEY) === 'true');
 
+    const getKbnVersionClass = () => {
+      // we assume that the version is valid and has the form 'X.X.X'
+      // strip out `SNAPSHOT` and reformat to 'X-X-X'
+      const formattedVersionClass = this.params.kibanaVersion
+        .replace(SNAPSHOT_REGEX, '')
```

Comment:
Inspired by https://github.com/elastic/kibana/blob/master/src/plugins/usage_collection/server/routes/stats/stats.ts#L176

Reply:
For SO mig, we're using `kibanaVersion.split('-')[0]`, but I guess both are fine, as I don't think we're using any other suffix than `-SNAPSHOT`